compose/musicbrainz-dev.yml

```diff
@@ -17,7 +17,7 @@ services:
     depends_on:
       - validator
   validator:
-    image: ghcr.io/validator/validator:21.7.0
+    image: ghcr.io/validator/validator@sha256:88641ddddc43c9414afe4903fb4b602aab2e81d87bae3a569ff79d611ffbcaf3
```

Comment:
Why using a SHA-256 hash instead of a semantic version?
Should `21.7.10`be used instead?

Reply:
Hi @yvanzo ! I did try to do that but was unable to resolve the issue. Upon reading https://github.com/validator/validator/pkgs/container/validator/3774673 I decided to use the SHA-256 hash and it worked. I am open to your opinions on this though.

Reply:
@akshaaatt, it looks like the issue was to use 21.7.**0** instead of 21.7.**10**:

```shell
$ sudo docker pull ghcr.io/validator/validator:21.7.0
Error response from daemon: manifest unknown
```

```shell
$ sudo docker pull ghcr.io/validator/validator:21.7.10
21.7.10: Pulling from validator/validator
...
Digest: sha256:88641ddddc43c9414afe4903fb4b602aab2e81d87bae3a569ff79d611ffbcaf3
Status: Downloaded newer image for ghcr.io/validator/validator:21.7.10
ghcr.io/validator/validator:21.7.10
```

Reply:
Such a silly mistake off! Thanks for the fix! I'll correct it asap.